src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+	
+	public String value;
+	protected boolean isPrivate; 
+	
+	public Contact(String value, boolean isPrivate){
+		this.value=value.trim();
+		this.isPrivate=isPrivate;
+}
```

Comment:
can have more methods in Contact instead of its subclasses so there is less repeated code e.g. toString